plugins/banners/get.js

```diff
@@ -13,6 +13,10 @@ module.exports = () => ({
         description: 'Get a single banner',
         notes: 'Return a banner record',
         tags: ['api', 'banners'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user']
```

Comment:
Banners with `GLOBAL` scope are currently being fetched without the user being authenticated.
![image](https://github.com/user-attachments/assets/c583569f-dd30-4b38-8e46-474276c71943)


This change makes it backward incompatible. We are unable fetch 'GLOBAL' banners when the user is not authenticated.
![image](https://github.com/user-attachments/assets/9fa851b7-342d-463d-9e12-fb4b30e60372)

Auth check may need to be done selectively based on the scope. `GLOBAL` can be accessed by anyone like before. But, `PIPELINE` should be accessed by a user with valid token.